crates/memory/src/game_engine/il2cpp/unity_items.rs

```diff
@@ -0,0 +1,50 @@
+use crate::process::{Error, Process};
+
+#[derive(Default, Debug)]
+pub struct UnityItems<T: UnityItem> {
```

Comment:
A minor note on this (depends if we need to implement other collections); maybe `UnityVec` or `UnityList` is more descriptive than `UnityItems`.

Reply:
I'll go and look up the actual internal type today and rename accordingly :)